docs/source/docker/getting_started/selection.rst

```diff
@@ -0,0 +1,521 @@
+                    "type": ...
+                },
+                "strategy": {
+                    "type": ...
+                }
+            },
+            ... more strategies
+        ]
+    }
+
+The variable :code:`n_samples` must be a positive integer specifying the absolute number of samples which should be selected.
+Alternative to :code:`n_samples`, you can also set :code:`proportion_samples` to set the number of samples to be selected relative to the input dataset size.
+E.g. set it to `0.1` to select 10% of all samples.
+Please set either one or the other. Setting both or none of them will cause an error.
+
+Each strategy is specified by a :code:`dictionary`, which is always made up of an :code:`input` and the actual :code:`strategy`.
+
+.. code-block:: python
+
+    {
+        "input": ...,
+        "strategy": ...
+    },
+
+
+Selection Input
+^^^^^^^^^^^^^^^^
+
+The input can be one of the following:
+
+.. tabs::
+
+    .. tab:: EMBEDDINGS
+
+        The `lightly OSS framework for self supervised learning <https://github.com/lightly-ai/lightly>`_ is used to compute the embeddings.
+        They are a vector of numbers for each sample.
+        
+        You can define embeddings as input using:
+
+        .. code-block:: python
+
+            "input": {
+                "type": "EMBEDDINGS"
+            }
+
+    .. tab:: SCORES
+
+        They are a scalar number for each element. They are **specified by the prediction task and the scorer**:
+
+        .. code-block:: python
+
+            # using your own predictions
+            "input": {
+                "type": "SCORES",
+                "task": "YOUR_TASK_NAME",
+                "score": "uncertainty_entropy"
+            }
+
+            # using the lightly pretagging model
+            "input": {
+                "type": "SCORES",
+                "task": "lightly_pretagging",
+                "score": "uncertainty_entropy"
+            }
+
+        You can specify one of the tasks you specified in your datasource, see :ref:`docker-datasource-predictions` for reference.
+        Alternatively, you can use **lightly_pretagging** as the task to use object detections created by the Lightly Worker itself.
+        See :ref:`docker-pretagging` for reference.
+
+        The supported score types are explained in :ref:`lightly-active-learning-scorers`.
+
+    .. tab:: PREDICTIONS
+
+        .. _worker-selection-predictions:
+
+        The class distribution probability vector of predictions can be used as well. Here, three cases have to be distinguished:
+
+            - **Image Classification** → The probability vector of each sample's prediction is used directly.
+
+            - **Object Detection** → The probability vectors of the class predictions of all objects in an image are summed up.
+
+            - **Object Detection** and using the :ref:`docker-object-level` → Each sample is a cropped object and has a single object prediction, whose probability vector is used.
```

Comment:
It is a bit special that we use arrows ("→") here as we don't use them anywhere else in the docs